lib/src/provider/asset_picker_provider.dart

```diff
@@ -490,6 +502,8 @@ class DefaultAssetPickerProvider
     final PathWrapper<AssetPathEntity> wrapper = _currentPath!;
     final int assetCount =
         wrapper.assetCount ?? await wrapper.path.assetCountAsync;
+    // If the picker was disposed (#492), stop fetching the assets
+    if (!mounted) return;
```

Comment:
I was wondering why this line doesn't hit the linter rule. It should be
```suggestion
    if (!mounted) {
      return;
    }
```

Reply:
If you're speaking about the `curly_braces_in_flow_control_structures` linter rule. There is an exception when the if statement and the body fit in one line (c.f. [curly_braces_in_flow_control_structures](https://dart.dev/tools/linter-rules/curly_braces_in_flow_control_structures)).

That's the reason, but if you prefer, i will add the curly braces.